docs/upgrading.rst

```diff
@@ -4,6 +4,15 @@ Upgrading
 .. toctree::
    :maxdepth: 1
 
+Upgrading to 4.0
+----------------
+
+Execution API Updates
+^^^^^^^^^^^^^^^^^^^^^
+:attr:`~.PreparedStatement.column_metadata` is renamed to :attr:`~.PreparedStatement.bind_metadata`
```

Comment:
Is this the correct way to reference the parameters? It looks [like this](https://github.com/datastax/python-driver/blob/ea00fa4020413656b6ecfe8982c0325ca2d1710e/docs/upgrading.rst#attrpreparedstatementcolumn_metadata-is-renamed-to-attrpreparedstatementbind_metadata) which doesn't look to good but the rest of the document is like that.

Reply:
Yeah, it's gonna look bad on github, unfortunately